omnicompletion not working

### Description

If I type \<c-x\>\<c-o\> in console or watches buffers I get errors

### Is it reproducible in vim?

Reproducible in Vim

### Works with clean config?

Yes

### Sample project works?

Reproducible with sample project

### Minimal Reproduction

List of steps to reproduce:

1. Download the following .zip file:

[test.zip](https://github.com/puremourning/vimspector/files/14295180/test.zip)

2. Comnpile test.cpp, i.e.
```bash
> g++ -g test.cpp -o test.exe
```
3. Run `vim ---clean Nu /path/to/vimspector/support/minimal_vimrc`
where "minimal_vimrc test.cpp " is 
```viml
set nocompatible
set runtimepath^=/Users/demichel/.vim/plugged/vimspector/
" Use <c-space> to trigger completion
filetype plugin indent on
set nohidden
function! Vis(...)
  if a:0 < 2
    echo '[ERROR] You must supply program name and arguments'
    return
  endif
  let l:allargs= join(a:000[1:])
  let l:dict={ 'configuration': 'fullvararg', 'Executable': a:1, 'CommandLineArgs': l:allargs }
  call vimspector#LaunchWithSettings( l:dict )
endfunction
command! -complete=file -nargs=+ D call Vis(<f-args>)
```
5. From command line execute:
```vim
D ./test.exe 3
```
6. go to "watches" buffer, type 'a' and type something and then <c-x><c-o>, you will get errors as shown
in the following screenshot:
<img width="2049" alt="Screenshot 2024-02-15 alle 11 08 24" src="https://github.com/puremourning/vimspector/assets/15322138/49959c21-5efc-4174-8bbf-8e606ee7533d">



### Expected Behaviour

no errors on omnicompletion

### Actual Behaviour

many errors

### Additional information

none

### Installation Type

Plugin Manager (git clone)

### Vimspector version

https://github.com/puremourning/vimspector/commit/7c0a89d9a91b7b3ece1db15661d85f372bbc9add

### Debug Info

```Text
Vimspector Debug Info
--------------------------------------------------------------------------------
ConnectionType: channel
Adapter: 
--------------------------------------------------------------------------------
{
  "command": [
    "/Users/demichel/.vim/plugged/vimspector/gadgets/macos/CodeLLDB/adapter/codelldb",
    "--port",
    "53987"
  ],
  "configuration": {
    "args": [],
    "cargo": {},
    "cwd": "/Users/demichel/VIM_STUFF/VIMSPECTOR",
    "env": {},
    "name": "lldb",
    "terminal": "integrated",
    "type": "lldb"
  },
  "name": "CodeLLDB",
  "port": "53987",
  "type": "CodeLLDB",
  "env": {},
  "cwd": "/Users/demichel/VIM_STUFF/VIMSPECTOR"
}
--------------------------------------------------------------------------------
Configuration: 
--------------------------------------------------------------------------------
{
  "adapter": "CodeLLDB",
  "breakpoints": {
    "exception": {
      "cpp_throw": "Y",
      "cpp_catch": "N"
    }
  },
  "configuration": {
    "request": "launch",
    "program": "./test.exe",
    "args": [
      "3"
    ],
    "stopAtEntry": true,
    "cwd": "/Users/demichel/VIM_STUFF/VIMSPECTOR"
  }
}
--------------------------------------------------------------------------------
API Prefix: 
Launch/Init: True / True
Workspace Root: /Users/demichel/VIM_STUFF/VIMSPECTOR
Launch Config: 
--------------------------------------------------------------------------------
{
  "args": [
    "3"
  ],
  "cargo": {},
  "cwd": "/Users/demichel/VIM_STUFF/VIMSPECTOR",
  "env": {},
  "name": "lldb",
  "terminal": "integrated",
  "type": "lldb",
  "request": "launch",
  "program": "./test.exe",
  "stopAtEntry": true
}
--------------------------------------------------------------------------------
Server Capabilities: 
--------------------------------------------------------------------------------
{
  "exceptionBreakpointFilters": [
    {
      "default": true,
      "filter": "cpp_throw",
      "label": "C++: on throw",
      "supportsCondition": true
    },
    {
      "default": false,
      "filter": "cpp_catch",
      "label": "C++: on catch",
      "supportsCondition": true
    }
  ],
  "supportTerminateDebuggee": true,
  "supportsCancelRequest": true,
  "supportsCompletionsRequest": true,
  "supportsConditionalBreakpoints": true,
  "supportsConfigurationDoneRequest": true,
  "supportsDataBreakpoints": true,
  "supportsDelayedStackTraceLoading": true,
  "supportsDisassembleRequest": true,
  "supportsEvaluateForHovers": true,
  "supportsExceptionFilterOptions": true,
  "supportsExceptionInfoRequest": true,
  "supportsFunctionBreakpoints": true,
  "supportsGotoTargetsRequest": true,
  "supportsHitConditionalBreakpoints": true,
  "supportsInstructionBreakpoints": true,
  "supportsLogPoints": true,
  "supportsReadMemoryRequest": true,
  "supportsSetVariable": true,
  "supportsSteppingGranularity": true,
  "supportsWriteMemoryRequest": true
}
--------------------------------------------------------------------------------
Line Breakpoints: 
--------------------------------------------------------------------------------
{}
--------------------------------------------------------------------------------
Func Breakpoints: 
--------------------------------------------------------------------------------
[]
--------------------------------------------------------------------------------
Ex Breakpoints: 
--------------------------------------------------------------------------------
{
  "filters": [
    "cpp_throw"
  ]
}
---------------------------------------------------------
```


### Log file

```Text
2024-02-15 11:08:02,958 - INFO - debug_session.py:128 - 0 - **** INITIALISING NEW VIMSPECTOR SESSION FOR ID 0 ****
2024-02-15 11:08:02,958 - INFO - debug_session.py:130 - 0 - API is: 
2024-02-15 11:08:02,959 - INFO - debug_session.py:131 - 0 - VIMSPECTOR_HOME = /Users/demichel/.vim/plugged/vimspector/python3/vimspector/../..
2024-02-15 11:08:02,959 - INFO - debug_session.py:132 - 0 - gadgetDir = /Users/demichel/.vim/plugged/vimspector/gadgets/macos
2024-02-15 11:08:02,959 - INFO - debug_session.py:226 - 0 - User requested start debug session with {'CommandLineArgs': '3', 'Executable': './test.exe', 'configuration': 'fullvararg'}
2024-02-15 11:08:02,959 - DEBUG - debug_session.py:187 - 0 - Reading configurations from: None
2024-02-15 11:08:02,959 - DEBUG - debug_session.py:187 - 0 - Reading configurations from: /Users/demichel/VIM_STUFF/VIMSPECTOR/.vimspector.json
2024-02-15 11:08:02,967 - DEBUG - debug_session.py:250 - 0 - Reading gadget config: /Users/demichel/.vim/plugged/vimspector/gadgets/macos/.gadgets.json
2024-02-15 11:08:02,976 - DEBUG - debug_session.py:250 - 0 - Reading gadget config: None
2024-02-15 11:08:02,976 - DEBUG - utils.py:633 - None - Returning Executable from the map for Executable with args ()
2024-02-15 11:08:02,976 - DEBUG - utils.py:633 - None - Returning CommandLineArgs from the map for CommandLineArgs with args ()
2024-02-15 11:08:02,976 - DEBUG - utils.py:684 - None - Put cwd into mapping for cwd with args ()
2024-02-15 11:08:02,976 - DEBUG - utils.py:633 - None - Returning cwd from the map for cwd with args ()
2024-02-15 11:08:02,976 - DEBUG - utils.py:633 - None - Returning gadgetDir from the map for gadgetDir with args ()
2024-02-15 11:08:02,976 - DEBUG - utils.py:684 - None - Put unusedLocalPort into mapping for unusedLocalPort with args ()
2024-02-15 11:08:02,976 - DEBUG - utils.py:633 - None - Returning unusedLocalPort from the map for unusedLocalPort with args ()
2024-02-15 11:08:02,976 - DEBUG - utils.py:633 - None - Returning workspaceRoot from the map for workspaceRoot with args ()
2024-02-15 11:08:02,976 - DEBUG - utils.py:633 - None - Returning unusedLocalPort from the map for unusedLocalPort with args ()
2024-02-15 11:08:02,976 - INFO - debug_session.py:483 - 0 - Configuration: {"adapter": "CodeLLDB", "breakpoints": {"exception": {"cpp_throw": "Y", "cpp_catch": "N"}}, "configuration": {"request": "launch", "program": "./test.exe", "args": ["3"], "stopAtEntry": true, "cwd": "/Users/demichel/VIM_STUFF/VIMSPECTOR"}}
2024-02-15 11:08:02,976 - INFO - debug_session.py:485 - 0 - Adapter: {"command": ["/Users/demichel/.vim/plugged/vimspector/gadgets/macos/CodeLLDB/adapter/codelldb", "--port", "53987"], "configuration": {"args": [], "cargo": {}, "cwd": "/Users/demichel/VIM_STUFF/VIMSPECTOR", "env": {}, "name": "lldb", "terminal": "integrated", "type": "lldb"}, "name": "CodeLLDB", "port": "53987", "type": "CodeLLDB"}
2024-02-15 11:08:02,977 - DEBUG - debug_session.py:1201 - 0 - min_width/height: 149/50, actual: 185/64 - result: horizontal
2024-02-15 11:08:02,986 - DEBUG - debug_session.py:1785 - 0 - LAUNCH!
2024-02-15 11:08:02,986 - INFO - debug_session.py:1400 - 0 - Starting debug adapter with: {"command": ["/Users/demichel/.vim/plugged/vimspector/gadgets/macos/CodeLLDB/adapter/codelldb", "--port", "53987"], "configuration": {"args": [], "cargo": {}, "cwd": "/Users/demichel/VIM_STUFF/VIMSPECTOR", "env": {}, "name": "lldb", "terminal": "integrated", "type": "lldb"}, "name": "CodeLLDB", "port": "53987", "type": "CodeLLDB"}
2024-02-15 11:08:02,986 - DEBUG - debug_session.py:1419 - 0 - Connection Type: channel
2024-02-15 11:08:03,154 - INFO - debug_session.py:1503 - 0 - Debug Adapter Started
2024-02-15 11:08:03,154 - DEBUG - debug_adapter_connection.py:205 - 0 - Sending Message: {"command": "initialize", "arguments": {"adapterID": "CodeLLDB", "clientID": "vimspector", "clientName": "vimspector", "linesStartAt1": true, "columnsStartAt1": true, "locale": "en_GB", "pathFormat": "path", "supportsVariableType": true, "supportsVariablePaging": false, "supportsRunInTerminalRequest": true, "supportsMemoryReferences": true, "supportsStartDebuggingRequest": true}, "seq": 0, "type": "request"}
2024-02-15 11:08:03,362 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 1, 'type': 'response', 'request_seq': 0, 'success': True, 'command': 'initialize', 'body': {'exceptionBreakpointFilters': [{'default': True, 'filter': 'cpp_throw', 'label': 'C++: on throw', 'supportsCondition': True}, {'default': False, 'filter': 'cpp_catch', 'label': 'C++: on catch', 'supportsCondition': True}], 'supportTerminateDebuggee': True, 'supportsCancelRequest': True, 'supportsCompletionsRequest': True, 'supportsConditionalBreakpoints': True, 'supportsConfigurationDoneRequest': True, 'supportsDataBreakpoints': True, 'supportsDelayedStackTraceLoading': True, 'supportsDisassembleRequest': True, 'supportsEvaluateForHovers': True, 'supportsExceptionFilterOptions': True, 'supportsExceptionInfoRequest': True, 'supportsFunctionBreakpoints': True, 'supportsGotoTargetsRequest': True, 'supportsHitConditionalBreakpoints': True, 'supportsInstructionBreakpoints': True, 'supportsLogPoints': True, 'supportsReadMemoryRequest': True, 'supportsSetVariable': True, 'supportsSteppingGranularity': True, 'supportsWriteMemoryRequest': True}}
2024-02-15 11:08:03,362 - DEBUG - debug_adapter_connection.py:205 - 0 - Sending Message: {"command": "launch", "arguments": {"args": ["3"], "cargo": {}, "cwd": "/Users/demichel/VIM_STUFF/VIMSPECTOR", "env": {}, "name": "lldb", "terminal": "integrated", "type": "lldb", "request": "launch", "program": "./test.exe", "stopAtEntry": true}, "seq": 1, "type": "request"}
2024-02-15 11:08:03,373 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 2, 'type': 'event', 'event': 'output', 'body': {'category': 'console', 'output': "Console is in 'commands' mode, prefix expressions with '?'.\n"}}
2024-02-15 11:08:03,373 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 3, 'type': 'event', 'event': 'initialized'}
2024-02-15 11:08:03,373 - DEBUG - debug_adapter_connection.py:205 - 0 - Sending Message: {"command": "setFunctionBreakpoints", "arguments": {"breakpoints": []}, "seq": 2, "type": "request"}
2024-02-15 11:08:03,374 - DEBUG - debug_adapter_connection.py:205 - 0 - Sending Message: {"command": "setExceptionBreakpoints", "arguments": {"filters": ["cpp_throw"]}, "seq": 3, "type": "request"}
2024-02-15 11:08:03,387 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 4, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': 'FFFFFFFFFFFFFFFF', 'id': 'FFFFFFFFFFFFFFFF', 'name': 'test.exe', 'path': '/Users/demichel/VIM_STUFF/VIMSPECTOR/test.exe', 'symbolFilePath': '/Users/demichel/VIM_STUFF/VIMSPECTOR/test.exe.dSYM/Contents/Resources/DWARF/test.exe', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,388 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 5, 'type': 'request', 'command': 'runInTerminal', 'arguments': {'args': ['/Users/demichel/.vim/plugged/vimspector/gadgets/macos/CodeLLDB/adapter/codelldb', 'terminal-agent', '--connect=53994'], 'cwd': '', 'kind': 'integrated', 'title': 'lldb'}}
2024-02-15 11:08:03,388 - DEBUG - debug_session.py:1964 - 0 - Defaulting working directory to /Users/demichel/VIM_STUFF/VIMSPECTOR
2024-02-15 11:08:03,392 - DEBUG - debug_adapter_connection.py:205 - 0 - Sending Message: {"seq": 4, "type": "response", "request_seq": 5, "command": "runInTerminal", "body": {"processId": 28284}, "success": true}
2024-02-15 11:08:03,393 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 6, 'type': 'response', 'request_seq': 2, 'success': True, 'command': 'setFunctionBreakpoints', 'body': {'breakpoints': []}}
2024-02-15 11:08:03,393 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 7, 'type': 'response', 'request_seq': 3, 'success': True, 'command': 'setExceptionBreakpoints'}
2024-02-15 11:08:03,393 - DEBUG - debug_adapter_connection.py:205 - 0 - Sending Message: {"command": "configurationDone", "seq": 5, "type": "request"}
2024-02-15 11:08:03,399 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 8, 'type': 'event', 'event': 'output', 'body': {'category': 'console', 'output': 'Launching: /Users/demichel/VIM_STUFF/VIMSPECTOR/test.exe 3\n'}}
2024-02-15 11:08:03,825 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 9, 'type': 'event', 'event': 'output', 'body': {'category': 'console', 'output': 'Launched process 28285\n'}}
2024-02-15 11:08:03,826 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 10, 'type': 'response', 'request_seq': 1, 'success': True, 'command': 'launch'}
2024-02-15 11:08:03,826 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 11, 'type': 'event', 'event': 'module', 'body': {'module': {'id': '100000000', 'name': ''}, 'reason': 'removed'}}
2024-02-15 11:08:03,826 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 12, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '100010000', 'id': '100010000', 'name': 'dyld', 'path': '/usr/lib/dyld', 'symbolFilePath': '/usr/lib/dyld', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,826 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 13, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '100000000', 'id': '100000000', 'name': 'test.exe', 'path': '/Users/demichel/VIM_STUFF/VIMSPECTOR/test.exe', 'symbolFilePath': '/Users/demichel/VIM_STUFF/VIMSPECTOR/test.exe.dSYM/Contents/Resources/DWARF/test.exe', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,827 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 14, 'type': 'event', 'event': 'continued', 'body': {'allThreadsContinued': True, 'threadId': 1926726}}
2024-02-15 11:08:03,827 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 15, 'type': 'response', 'request_seq': 5, 'success': True, 'command': 'configurationDone'}
2024-02-15 11:08:03,827 - DEBUG - debug_adapter_connection.py:205 - 0 - Sending Message: {"command": "threads", "seq": 6, "type": "request"}
2024-02-15 11:08:03,833 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 16, 'type': 'response', 'request_seq': 6, 'success': True, 'command': 'threads', 'body': {'threads': [{'id': 1926726, 'name': '1: tid=1926726'}]}}
2024-02-15 11:08:03,833 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 17, 'type': 'event', 'event': 'module', 'body': {'module': {'id': 'FFFFFFFFFFFFFFFF', 'name': ''}, 'reason': 'removed'}}
2024-02-15 11:08:03,833 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 18, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DBE0000', 'id': '7FF80DBE0000', 'name': 'dyld', 'path': '/usr/lib/dyld', 'symbolFilePath': '/usr/lib/dyld', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,833 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 19, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '100000000', 'id': '100000000', 'name': 'test.exe', 'path': '/Users/demichel/VIM_STUFF/VIMSPECTOR/test.exe', 'symbolFilePath': '/Users/demichel/VIM_STUFF/VIMSPECTOR/test.exe.dSYM/Contents/Resources/DWARF/test.exe', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,919 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 20, 'type': 'event', 'event': 'breakpoint', 'body': {'breakpoint': {'id': 1, 'message': 'Resolved locations: 2', 'verified': True}, 'reason': 'changed'}}
2024-02-15 11:08:03,919 - WARNING - breakpoints.py:561 - 0 - Unexpected update to breakpoint with id 1:breakpiont not found. {'id': 1, 'message': 'Resolved locations: 2', 'verified': True}
2024-02-15 11:08:03,919 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 21, 'type': 'event', 'event': 'breakpoint', 'body': {'breakpoint': {'id': 1, 'message': 'Resolved locations: 2', 'verified': True}, 'reason': 'changed'}}
2024-02-15 11:08:03,919 - WARNING - breakpoints.py:561 - 0 - Unexpected update to breakpoint with id 1:breakpiont not found. {'id': 1, 'message': 'Resolved locations: 2', 'verified': True}
2024-02-15 11:08:03,920 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 22, 'type': 'event', 'event': 'breakpoint', 'body': {'breakpoint': {'id': 1, 'message': 'Resolved locations: 2', 'verified': True}, 'reason': 'changed'}}
2024-02-15 11:08:03,920 - WARNING - breakpoints.py:561 - 0 - Unexpected update to breakpoint with id 1:breakpiont not found. {'id': 1, 'message': 'Resolved locations: 2', 'verified': True}
2024-02-15 11:08:03,932 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 23, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '100A04000', 'id': '100A04000', 'name': 'libstdc++.6.dylib', 'path': '/usr/local/Cellar/gcc/13.2.0/lib/gcc/current/libstdc++.6.dylib', 'symbolFilePath': '/usr/local/Cellar/gcc/13.2.0/lib/gcc/current/libstdc++.6.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,933 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 24, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '10003E000', 'id': '10003E000', 'name': 'libgcc_s.1.1.dylib', 'path': '/usr/local/Cellar/gcc/13.2.0/lib/gcc/current/libgcc_s.1.1.dylib', 'symbolFilePath': '/usr/local/Cellar/gcc/13.2.0/lib/gcc/current/libgcc_s.1.1.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,933 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 25, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE72000', 'id': '7FF81AE72000', 'name': 'libiconv.2.dylib', 'path': '/usr/lib/libiconv.2.dylib', 'symbolFilePath': '/usr/lib/libiconv.2.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,933 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 26, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81ADC2000', 'id': '7FF81ADC2000', 'name': 'libcharset.1.dylib', 'path': '/usr/lib/libcharset.1.dylib', 'symbolFilePath': '/usr/lib/libcharset.1.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,934 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 27, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE35000', 'id': '7FF81AE35000', 'name': 'libSystem.B.dylib', 'path': '/usr/lib/libSystem.B.dylib', 'symbolFilePath': '/usr/lib/libSystem.B.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,934 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 28, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE2F000', 'id': '7FF81AE2F000', 'name': 'libcache.dylib', 'path': '/usr/lib/system/libcache.dylib', 'symbolFilePath': '/usr/lib/system/libcache.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,934 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 29, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81ADEA000', 'id': '7FF81ADEA000', 'name': 'libcommonCrypto.dylib', 'path': '/usr/lib/system/libcommonCrypto.dylib', 'symbolFilePath': '/usr/lib/system/libcommonCrypto.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,934 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 30, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE13000', 'id': '7FF81AE13000', 'name': 'libcompiler_rt.dylib', 'path': '/usr/lib/system/libcompiler_rt.dylib', 'symbolFilePath': '/usr/lib/system/libcompiler_rt.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,934 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 31, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE09000', 'id': '7FF81AE09000', 'name': 'libcopyfile.dylib', 'path': '/usr/lib/system/libcopyfile.dylib', 'symbolFilePath': '/usr/lib/system/libcopyfile.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,934 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 32, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DCDF000', 'id': '7FF80DCDF000', 'name': 'libcorecrypto.dylib', 'path': '/usr/lib/system/libcorecrypto.dylib', 'symbolFilePath': '/usr/lib/system/libcorecrypto.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,935 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 33, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DDC6000', 'id': '7FF80DDC6000', 'name': 'libdispatch.dylib', 'path': '/usr/lib/system/libdispatch.dylib', 'symbolFilePath': '/usr/lib/system/libdispatch.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,935 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 34, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DF77000', 'id': '7FF80DF77000', 'name': 'libdyld.dylib', 'path': '/usr/lib/system/libdyld.dylib', 'symbolFilePath': '/usr/lib/system/libdyld.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,935 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 35, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE25000', 'id': '7FF81AE25000', 'name': 'libkeymgr.dylib', 'path': '/usr/lib/system/libkeymgr.dylib', 'symbolFilePath': '/usr/lib/system/libkeymgr.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,935 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 36, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81ADC6000', 'id': '7FF81ADC6000', 'name': 'libmacho.dylib', 'path': '/usr/lib/system/libmacho.dylib', 'symbolFilePath': '/usr/lib/system/libmacho.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,935 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 37, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81A259000', 'id': '7FF81A259000', 'name': 'libquarantine.dylib', 'path': '/usr/lib/system/libquarantine.dylib', 'symbolFilePath': '/usr/lib/system/libquarantine.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,936 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 38, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE23000', 'id': '7FF81AE23000', 'name': 'libremovefile.dylib', 'path': '/usr/lib/system/libremovefile.dylib', 'symbolFilePath': '/usr/lib/system/libremovefile.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,936 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 39, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81359D000', 'id': '7FF81359D000', 'name': 'libsystem_asl.dylib', 'path': '/usr/lib/system/libsystem_asl.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_asl.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,936 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 40, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DC7D000', 'id': '7FF80DC7D000', 'name': 'libsystem_blocks.dylib', 'path': '/usr/lib/system/libsystem_blocks.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_blocks.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,936 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 41, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DE10000', 'id': '7FF80DE10000', 'name': 'libsystem_c.dylib', 'path': '/usr/lib/system/libsystem_c.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_c.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,936 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 42, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE1B000', 'id': '7FF81AE1B000', 'name': 'libsystem_collections.dylib', 'path': '/usr/lib/system/libsystem_collections.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_collections.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,936 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 43, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF819377000', 'id': '7FF819377000', 'name': 'libsystem_configuration.dylib', 'path': '/usr/lib/system/libsystem_configuration.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_configuration.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,936 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 44, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF8183E5000', 'id': '7FF8183E5000', 'name': 'libsystem_containermanager.dylib', 'path': '/usr/lib/system/libsystem_containermanager.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_containermanager.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 45, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AA1E000', 'id': '7FF81AA1E000', 'name': 'libsystem_coreservices.dylib', 'path': '/usr/lib/system/libsystem_coreservices.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_coreservices.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 46, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81116F000', 'id': '7FF81116F000', 'name': 'libsystem_darwin.dylib', 'path': '/usr/lib/system/libsystem_darwin.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_darwin.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 47, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FFC24CCA000', 'id': '7FFC24CCA000', 'name': 'libsystem_darwindirectory.dylib', 'path': '/usr/lib/system/libsystem_darwindirectory.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_darwindirectory.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 48, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE26000', 'id': '7FF81AE26000', 'name': 'libsystem_dnssd.dylib', 'path': '/usr/lib/system/libsystem_dnssd.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_dnssd.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 49, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DE0D000', 'id': '7FF80DE0D000', 'name': 'libsystem_featureflags.dylib', 'path': '/usr/lib/system/libsystem_featureflags.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_featureflags.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 50, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DFA6000', 'id': '7FF80DFA6000', 'name': 'libsystem_info.dylib', 'path': '/usr/lib/system/libsystem_info.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_info.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 51, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AD5E000', 'id': '7FF81AD5E000', 'name': 'libsystem_m.dylib', 'path': '/usr/lib/system/libsystem_m.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_m.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 52, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DD8D000', 'id': '7FF80DD8D000', 'name': 'libsystem_malloc.dylib', 'path': '/usr/lib/system/libsystem_malloc.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_malloc.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,937 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 53, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF813513000', 'id': '7FF813513000', 'name': 'libsystem_networkextension.dylib', 'path': '/usr/lib/system/libsystem_networkextension.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_networkextension.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,938 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 54, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF8115B9000', 'id': '7FF8115B9000', 'name': 'libsystem_notify.dylib', 'path': '/usr/lib/system/libsystem_notify.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_notify.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,938 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 55, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81937B000', 'id': '7FF81937B000', 'name': 'libsystem_sandbox.dylib', 'path': '/usr/lib/system/libsystem_sandbox.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_sandbox.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,938 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 56, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE20000', 'id': '7FF81AE20000', 'name': 'libsystem_secinit.dylib', 'path': '/usr/lib/system/libsystem_secinit.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_secinit.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,938 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 57, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DF30000', 'id': '7FF80DF30000', 'name': 'libsystem_kernel.dylib', 'path': '/usr/lib/system/libsystem_kernel.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_kernel.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,938 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 58, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DF9C000', 'id': '7FF80DF9C000', 'name': 'libsystem_platform.dylib', 'path': '/usr/lib/system/libsystem_platform.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_platform.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,938 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 59, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DF6B000', 'id': '7FF80DF6B000', 'name': 'libsystem_pthread.dylib', 'path': '/usr/lib/system/libsystem_pthread.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_pthread.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,939 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 60, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF815136000', 'id': '7FF815136000', 'name': 'libsystem_symptoms.dylib', 'path': '/usr/lib/system/libsystem_symptoms.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_symptoms.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,939 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 61, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DCC5000', 'id': '7FF80DCC5000', 'name': 'libsystem_trace.dylib', 'path': '/usr/lib/system/libsystem_trace.dylib', 'symbolFilePath': '/usr/lib/system/libsystem_trace.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,939 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 62, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81ADF6000', 'id': '7FF81ADF6000', 'name': 'libunwind.dylib', 'path': '/usr/lib/system/libunwind.dylib', 'symbolFilePath': '/usr/lib/system/libunwind.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,939 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 63, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DC81000', 'id': '7FF80DC81000', 'name': 'libxpc.dylib', 'path': '/usr/lib/system/libxpc.dylib', 'symbolFilePath': '/usr/lib/system/libxpc.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,939 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 64, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DF17000', 'id': '7FF80DF17000', 'name': 'libc++abi.dylib', 'path': '/usr/lib/libc++abi.dylib', 'symbolFilePath': '/usr/lib/libc++abi.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,939 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 65, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DB9E000', 'id': '7FF80DB9E000', 'name': 'libobjc.A.dylib', 'path': '/usr/lib/libobjc.A.dylib', 'symbolFilePath': '/usr/lib/libobjc.A.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,940 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 66, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF81AE01000', 'id': '7FF81AE01000', 'name': 'liboah.dylib', 'path': '/usr/lib/liboah.dylib', 'symbolFilePath': '/usr/lib/liboah.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,940 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 67, 'type': 'event', 'event': 'module', 'body': {'module': {'addressRange': '7FF80DE98000', 'id': '7FF80DE98000', 'name': 'libc++.1.dylib', 'path': '/usr/lib/libc++.1.dylib', 'symbolFilePath': '/usr/lib/libc++.1.dylib', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2024-02-15 11:08:03,942 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 68, 'type': 'event', 'event': 'output', 'body': {'category': 'console', 'output': 'Process exited with code 0.\n'}}
2024-02-15 11:08:03,943 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 69, 'type': 'event', 'event': 'exited', 'body': {'exitCode': 0}}
2024-02-15 11:08:03,943 - DEBUG - debug_adapter_connection.py:266 - 0 - Message received: {'seq': 70, 'type': 'event', 'event': 'terminated', 'body': {}}
```


### Vim version

```Text
VIM - Vi IMproved 9.1 (2024 Jan 02, compilato Feb 12 2024 19:28:01)
Versione macOS - x86_64
Patch incluse: 1-100
Compilato da Homebrew
Versione gigante senza GUI.  Funzionalità incluse (+) o escluse (-):
+acl               +file_in_path      +mouse_urxvt       -tag_any_white
+arabic            +find_in_path      +mouse_xterm       -tcl
+autocmd           +float             +multi_byte        +termguicolors
+autochdir         +folding           +multi_lang        +terminal
-autoservername    -footer            -mzscheme          +terminfo
-balloon_eval      +fork()            +netbeans_intg     +termresponse
+balloon_eval_term +gettext           +num64             +textobjects
-browse            -hangul_input      +packages          +textprop
++builtin_terms    +iconv             +path_extra        +timers
+byte_offset       +insert_expand     +perl              +title
+channel           +ipv6              +persistent_undo   -toolbar
+cindent           +job               +popupwin          +user_commands
-clientserver      +jumplist          +postscript        +vartabs
+clipboard         +keymap            +printer           +vertsplit
+cmdline_compl     +lambda            +profile           +vim9script
+cmdline_hist      +langmap           -python            +viminfo
+cmdline_info      +libcall           +python3           +virtualedit
+comments          +linebreak         +quickfix          +visual
+conceal           +lispindent        +reltime           +visualextra
+cryptv            +listcmds          +rightleft         +vreplace
+cscope            +localmap          +ruby              +wildignore
+cursorbind        +lua               +scrollbind        +wildmenu
+cursorshape       +menu              +signs             +windows
+dialog_con        +mksession         +smartindent       +writebackup
+diff              +modify_fname      +sodium            -X11
+digraphs          +mouse             +sound             -xattr
-dnd               -mouseshape        +spell             -xfontset
-ebcdic            +mouse_dec         +startuptime       -xim
+emacs_tags        -mouse_gpm         +statusline        -xpm
+eval              -mouse_jsbterm     -sun_workshop      -xsmp
+ex_extra          +mouse_netterm     +syntax            -xterm_clipboard
+extra_search      +mouse_sgr         +tag_binary        -xterm_save
-farsi             -mouse_sysmouse    -tag_old_static
   file vimrc di sistema: "$VIM/vimrc"
       file vimrc utente: "$HOME/.vimrc"
    II file vimrc utente: "~/.vim/vimrc"
        file exrc utente: "$HOME/.exrc"
        file dei default: "$VIMRUNTIME/defaults.vim"
         $VIM di riserva: "/usr/local/share/vim"
Compilazione: clang -c -I. -Iproto -DHAVE_CONFIG_H -DMACOS_X -DMACOS_X_DARWIN -g -O2 -D_REENTRANT -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1
Link: clang -L/usr/local/lib -o vim -lm -lncurses -lsodium -liconv -lintl -framework AppKit -L/usr/local/opt/lua/lib -llua5.4 -mmacosx-version-min=14.2 -fstack-protector-strong -L/usr/local/lib -L/usr/local/opt/perl/lib/perl5/5.38/darwin-thread-multi-2level/CORE -lperl -L/usr/local/opt/python@3.12/Frameworks/Python.framework/Versions/3.12/lib/python3.12/config-3.12-darwin -lpython3.12 -framework CoreFoundation -lruby.3.3 -L/usr/local/Cellar/ruby/3.3.0/lib
```


### Python version

_No response_

### Neovim diagnostics

_No response_

### Operating System

Darwin piro.phys.uniroma1.it 23.3.0 Darwin Kernel Version 23.3.0: Wed Dec 20 21:28:58 PST 2023; root:xnu-10002.81.5~7/RELEASE_X86_64 x86_64

### Declaration

- [X] I have read and understood [CONTRIBUTING.md](https://github.com/puremourning/vimspector/blob/master/CONTRIBUTING.md#diagnostics)
- [X] I have read and understood the [CODE_OF_CONDUCT.md](https://github.com/puremourning/vimspector/blob/master/CODE_OF_CONDUCT.md)

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -114,7 +114,9 @@ jobs:
       matrix:
         runtime:
           - vim
-          - nvim
+          # - nvim ; MacOS in GHA is so slow, this seems to cause lots of
+          # flakiness for neovim and I have like 0 patience with trying to
+          # understand why.
     steps:
     - uses: actions/checkout@v3
 

--- a/.mergify.yml
+++ b/.mergify.yml
@@ -9,7 +9,7 @@ queue_rules:
       - status-success=Linux (vim)
       - status-success=Linux (nvim)
       - status-success=MacOS (vim)
-      - status-success=MacOS (nvim)
+      #- status-success=MacOS (nvim)
 
 pull_request_rules:
   - name: Merge owner PR when all checks passed
@@ -26,7 +26,7 @@ pull_request_rules:
       - status-success=Linux (vim)
       - status-success=Linux (nvim)
       - status-success=MacOS (vim)
-      - status-success=MacOS (nvim)
+      #- status-success=MacOS (nvim)
     actions: &merge-actions
       queue:
         method: merge
@@ -61,7 +61,7 @@ pull_request_rules:
       - status-success=Linux (vim)
       - status-success=Linux (nvim)
       - status-success=MacOS (vim)
-      - status-success=MacOS (nvim)
+      #- status-success=MacOS (nvim)
     actions:
       <<: *merge-actions
       comment:

--- a/python3/vimspector/variables.py
+++ b/python3/vimspector/variables.py
@@ -176,6 +176,10 @@ class Watch:
     elif self.connection != connection:
       return
 
+    if frame is None:
+      # Evaluation in a context where there is no current frame.
+      return
+
     self.expression[ 'frameId' ] = frame[ 'id' ]
 
   @staticmethod

--- a/tests/variables.test.vim
+++ b/tests/variables.test.vim
@@ -19,15 +19,15 @@ function! ConsoleBufferName()
 endfunction
 
 function! s:StartDebugging( ... )
-  if a:0 == 0
-    let config = #{
-          \   fn: s:fn,
-          \   line: 23,
-          \   col: 1,
-          \   launch: #{ configuration: 'run' }
-          \ }
-  else
-    let config = a:1
+  let config = #{
+        \   fn: s:fn,
+        \   line: 23,
+        \   col: 1,
+        \   launch: #{ configuration: 'run' }
+        \ }
+
+  if a:0 > 0
+    call extend( config, a:1 )
   endif
 
   execute 'edit' config.fn
@@ -1257,6 +1257,46 @@ EOF
         \       ' + another_test: ',
         \     ],
         \     getbufline( winbufnr( g:vimspector_session_windows.eval ),
+        \                 1,
+        \                 '$' )
+        \   )
+        \ } )
+
+  call vimspector#test#setup#Reset()
+  %bwipe!
+endfunction
+
+function! Test_WatchAfterExit()
+  call s:StartDebugging( #{
+        \ col: v:null,
+        \ line: 3,
+        \ fn: 'testdata/cpp/simple/tiny.c',
+        \ launch: #{ configuration: 'CodeLLDB' }
+        \ } )
+
+  call vimspector#Continue()
+  call WaitForAssert( {->
+        \   AssertMatchList(
+        \     [
+        \         '+ Thread [0-9]\+: .* (terminated)',
+        \     ],
+        \     GetBufLine( winbufnr( g:vimspector_session_windows.stack_trace ),
+        \                 1,
+        \                 '$' )
+        \   )
+        \ } )
+
+  " Add a wtch
+  call vimspector#AddWatch( 'res' )
+
+  call WaitForAssert( {->
+        \   AssertMatchList(
+        \     [
+        \       'Watches: ----',
+        \       'Expression: res',
+        \       ' \*- Result: .*',
+        \     ],
+        \     getbufline( winbufnr( g:vimspector_session_windows.watches ),
         \                 1,
         \                 '$' )
         \   )

